packages/react-shopify-app-bridge/README.md

```diff
@@ -59,6 +59,9 @@ NOTE: This example is very similar to that found in [@gadgetinc/react](https://g
 `src/api.ts`
 
 ```typescript
+// replace `my-app-slug` with your app slug from your Gadget app's domain
+import { Client, BrowserSessionStorageType } from "@gadget-client/my-app-slug";
```

Comment:
Need this import statement to know what `Client` is below, and it references the `BrowserSessionStorageType` constant